src/config/ethereum.ts

```diff
@@ -4,6 +4,12 @@ const appChainId = process.env.REACT_APP_CHAIN as EthereumChainId;
 const appEnv = process.env.REACT_APP_ENV as Networks;
 const infuraId = process.env.REACT_APP_INFURA_ID;
 
+const vegaTokenAddress = process.env.NETWORK_TOKEN_ADDRESS || "0xF0598Cd16FA3bf4c34052923cBE2D34028da0c69"
```

Comment:
I find it's not particularly obvious that these are to be used by the validator network config. Do they need to be globally defined?